docs/resources/context_attachment.md

```diff
@@ -39,7 +39,7 @@ resource "spacelift_context_attachment" "attachment" {
 
 - **id** (String) The ID of this resource.
 - **module_id** (String) ID of the module to attach the context to
-- **priority** (Number) Priority of the context attachment, used in case of conflicts
+- **priority** (Number) Priority of the context attachment. All the contexts attached to a stack are sorted by priority (lowest first), though values don't need to be unique. This ordering establishes precedence rules between contexts should there be a conflict and multiple contexts defined the same value.
```

Comment:
All the contexts attached to a stack are sorted by priority (lowest first), though values don't need to be unique. This ordering establishes precedence rules between contexts should there be a conflict and multiple contexts **define** the same value.

Reply:
If you consider that this is incorrect then I will update [the documentation](https://docs.spacelift.io/concepts/configuration/context#a-note-on-priority) because [I stole that bit from it](https://github.com/spacelift-io/terraform-provider-spacelift/pull/282/files).